web-registry/src/pages/CreditClassDetails/tables/CreditClassDetails.TableTabs.tsx

```diff
@@ -0,0 +1,51 @@
+import { Box, useTheme } from '@mui/material';
+
+import { IconTabs } from 'web-components/lib/components/tabs/IconTabs';
+import {
+  containerPaddingX,
+  containerStyles,
+} from 'web-components/lib/styles/container';
+
+import { getCreditClassDetailsTabs } from './CreditClassDetails.TableTabs.config';
+import { CreditClassDetailsTableTabsProps } from './CreditClassDetails.TableTabs.types';
+
+export const CreditClassDetailsTableTabs = (
+  props: CreditClassDetailsTableTabsProps,
+) => {
+  const tabs = getCreditClassDetailsTabs(props);
+  const theme = useTheme();
+
+  return tabs.length > 0 ? (
+    <Box
+      sx={{
+        ...containerStyles,
+        ...containerPaddingX,
+        paddingTop: { xs: 20, sm: 27.5 },
+        paddingBottom: { xs: 20, sm: 25 },
+      }}
+    >
+      <IconTabs
+        tabs={tabs}
+        size="xl"
+        sxs={{
+          tab: {
+            outer: {
+              border: `1px solid ${theme.palette.info.light}`,
+              borderBottom: 0,
+              borderRadius: '8px 8px 0 0',
+              backgroundColor: 'primary.main',
+            },
+            innerContainer: {
+              paddingTop: 8.5,
+              paddingBottom: 6.25,
+              '&:first-child': {
+                marginLeft: { xs: 4, sm: 8.5 },
+              },
+            },
+          },
+          panel: { inner: { p: 0 } },
+        }}
+      />
+    </Box>
+  ) : null;
```

Comment:
we should extract this into a reusable component which takes some `IconTabProps[]` as props, so it can be used for both credit class and project
right now, this is duplicating what we have already in `ProjectDetailsTableTabs`

Reply:
https://github.com/regen-network/regen-web/pull/1985/commits/d29272a689758cce82e76f6f41a0ef7eb0510cc6